### PR TITLE
build: make `prod-build` script app-specific

### PR DESCRIPTION
**Pull request by eventualbuddha**
This allows building a single app at a time, effectively replacing much of `setup-app.sh` in `vxsuite-complete-system`. It's also in TypeScript instead of bash.

It puts the config for which modules are required by a given app in `package.json` for each app. This may or may want to to be its own file rather than piggybacking on `package.json`.

### CODE REVIEW

File: apps/bas/Makefile [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # a phony dependency that can be used as a dependency to force builds
 FORCE:
 
+install:
+	# sudo apt install -y DEPS
```
Comment (carolinemodic) [inline apps/bas/Makefile:6]: Why is this commented out? Also same question for the other apps with this haha.

Comment (eventualbuddha) [reply apps/bas/Makefile:6]: These don’t actually need to install anything, but the build script runs the `install` target in all the apps.

Comment (benadida) [reply apps/bas/Makefile:6]: should we say that in the comment then, for posterity?

File: apps/bas/package.json [hunk 2]
```diff
@@ -101,5 +101,11 @@
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.10.0"
+  },
+  "vx": {
```
Comment (benadida) [inline apps/bas/package.json:105]: I kinda like putting this in `package.json`

**Review by benadida — APPROVED** [review]

File: apps/bas/package.json (hunk 2 above)
Comment (eventualbuddha) [reply apps/bas/package.json:105]: The downside is that it doesn't work with non-nodejs apps/packages.

**Pull request merged**